app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better if we implemented an `after_save` call back for this method. So that the method is automatically called whenever a comment is saved. It would save us the stress of having to call it ourselves.

  ```
  after_save :update_comments_counter

  ...

  private
  
  def update_comments_counter
    post.increment!(:comments_counter)
  end
  ```